### PR TITLE
Create `bin/php-gameboy` executable

### PR DESCRIPTION
**Pull request by henriquemoody**

### CODE REVIEW

**Comment by gabrielrcouto** [comment]
Do you have any solution for Windows users @henriquemoody?

**Comment by henriquemoody** [comment]
Yes, they gotta start using Linux/Mac. :stuck_out_tongue: 

However, they can always run the executable with PHP command, like:

``` bash
php bin/php-gameboy /path/to/the/file.rom
```

If you take a look at the repositories of PHPUnit, Behat, Composer, etc, they use the same approach.

File: bin/php-gameboy [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__.'/../boot.php';
```
Comment (henriquemoody) [inline bin/php-gameboy:4]: Actually I'm favor of moving the content of `boot.php` to this file, but that's your call.

**Comment by gabrielrcouto** [comment]
Nice :smile:

**Pull request merged**